noir-projects/aztec-nr/aztec/src/macros/storage.nr

```diff
@@ -15,6 +15,18 @@ pub comptime mut global STORAGE_LAYOUT_NAME: UHashMap<Module, Quoted, BuildHashe
 /// The contract's storage is accessed via the `storage` variable, which will will automatically be made available in
 /// all functions as an instance of the struct this macro was applied to.
 pub comptime fn storage(s: TypeDefinition) -> Quoted {
```

Comment:
Would change this sentence:

"Only a single struct in the entire contract should have this macro (or `storage_no_init`) applied to it."

to
"Only a single struct in the entire contract can have this macro (or storage_no_init) applied to it and the struct has to be called Storage."